src/Sulu/Bundle/MediaBundle/Tests/Functional/Controller/MediaControllerTest.php

```diff
@@ -142,68 +169,58 @@ protected function setUpMedia()
         $file->addFileVersion($fileVersion);
 
         $media->addFile($file);
+        $media->setCollection($this->collection);
 
-        // Setup Collection
-        $collection = new Collection();
-
-        $this->setUpCollection($collection);
-
-        $media->setCollection($collection);
-
-        $this->em->persist($tag1);
-        $this->em->persist($tag2);
         $this->em->persist($media);
         $this->em->persist($file);
         $this->em->persist($fileVersionMeta);
         $this->em->persist($fileVersion);
-        $this->em->persist($mediaType);
-        $this->em->persist($imageType);
-        $this->em->persist($videoType);
 
         $this->em->flush();
+
+        return $media;
     }
 
-    protected function setUpCollection(&$collection)
+    protected function setUpCollection()
     {
+        $this->collection = new Collection();
         $style = array(
             'type' => 'circle',
             'color' => '#ffcc00'
         );
 
-        $collection->setStyle(json_encode($style));
```

Comment:
fyi doctrine can handle this natively now: http://doctrine-dbal.readthedocs.org/en/latest/reference/types.html#json-array /cc @alexander-schranz